(node:21) ExperimentalWarning: CommonJS module /home/container/tmp/node-v23.1.0-linux-x64/lib/node_modules/npm/node_modules/debug/src/node.js is loading ES Module /home/container/tmp/node-v23.1.0-linux-x64/lib/node_modules/npm/node_modules/supports-color/index.js using require().
Support for loading ES Module in require() is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

up to date, audited 167 packages in 1s

27 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

> discord-bot@1.0.0 start
> node index.js

ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± activedevbadge
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± admin-data
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± create_ticket_channels
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± create-welcome
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± force-close-tickets
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± help
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± ping
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± reset-tickets
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± setup_attendance
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± update-commands
ØªÙ… ØªØ­Ù…ÙŠÙ„ Ø§Ù„Ø£Ù…Ø± vac
ØªÙ… ØªØ³Ø¬ÙŠÙ„ Ø§Ù„Ø¯Ø®ÙˆÙ„ Ø¨Ù†Ø¬Ø§Ø­
ØªÙ… Ø¥Ø¹Ø¯Ø§Ø¯ Ø§Ù„Ø¨ÙˆØª Ø¨Ù†Ø¬Ø§Ø­
[32minfo[39m: ØªÙ… ØªØ³Ø¬ÙŠÙ„ Ø§Ù„Ø¯Ø®ÙˆÙ„ ÙƒÙ€ ArabPast Bot#3173 {"timestamp":"2024-11-04T22:10:05.260Z"}
ØªÙ… ØªØ³Ø¬ÙŠÙ„ Ø§Ù„Ø¯Ø®ÙˆÙ„ ÙƒÙ€ ArabPast Bot#3173
[32minfo[39m: ØªÙ… Ø§Ù„Ø§ØªØµØ§Ù„ Ø¨Ù‚Ø§Ø¹Ø¯Ø© Ø§Ù„Ø¨ÙŠØ§Ù†Ø§Øª MongoDB {"timestamp":"2024-11-04T22:10:05.702Z"}
Ø¨Ø¯Ø¡ ØªØ­Ø¯ÙŠØ« 11 Ù…Ù† Ø§Ù„Ø£ÙˆØ§Ù…Ø±.
âœ… ØªÙ… ØªØ­Ø¯ÙŠØ« 11 Ù…Ù† Ø§Ù„Ø£ÙˆØ§Ù…Ø± Ø¨Ù†Ø¬Ø§Ø­.
[32minfo[39m: Generating daily statistics... {"timestamp":"2024-11-05T00:00:00.555Z"}
[32minfo[39m: Button pressed: check_in {"timestamp":"2024-11-05T01:23:32.852Z"}
Starting check-in process for user: 803468870224052254
[32minfo[39m: Button pressed: check_out {"timestamp":"2024-11-05T01:24:54.043Z"}
[31merror[39m: Error in check-out: Error in check-out: {"critical":false,"stack":"ReferenceError: Attendance is not defined\n    at handleCheckOut (/home/container/index.js:1017:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Client.<anonymous> (/home/container/index.js:362:13)","timestamp":"2024-11-05T01:24:54.261Z"}
[32minfo[39m: Button pressed: check_out {"timestamp":"2024-11-05T01:24:57.659Z"}
[31merror[39m: Error in check-out: Error in check-out: {"critical":false,"stack":"ReferenceError: Attendance is not defined\n    at handleCheckOut (/home/container/index.js:1017:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Client.<anonymous> (/home/container/index.js:362:13)","timestamp":"2024-11-05T01:24:57.876Z"}
[32minfo[39m: Button pressed: check_out {"timestamp":"2024-11-05T01:25:06.391Z"}
[31merror[39m: Error in check-out: Error in check-out: {"critical":false,"stack":"ReferenceError: Attendance is not defined\n    at handleCheckOut (/home/container/index.js:1017:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Client.<anonymous> (/home/container/index.js:362:13)","timestamp":"2024-11-05T01:25:06.594Z"}
[32minfo[39m: Button pressed: check_in {"timestamp":"2024-11-05T06:28:41.596Z"}
Starting check-in process for user: 743432232529559684
[32minfo[39m: Button pressed: check_out {"timestamp":"2024-11-05T06:28:57.220Z"}
[31merror[39m: Error in check-out: Error in check-out: {"critical":false,"stack":"ReferenceError: Attendance is not defined\n    at handleCheckOut (/home/container/index.js:1017:24)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Client.<anonymous> (/home/container/index.js:362:13)","timestamp":"2024-11-05T06:28:57.445Z"}
